README.md

```diff
@@ -74,86 +137,72 @@ Dockerfiles (see [Dockerfile][here_container_dockerfile]).
 Please read more on the repo tool usage from
 [its official documentation][repo_tool].
 
-## Linux Requirements
+### Installation
 
-The implementation is based on DBus. The C binding used to access DBus is sdbus
-from systemd library so even if systemd is not required to be running, the
-libsystemd must be available.
+#### Sink service
 
-Systemd version must be higher or equal to *221*. You can check it with:
-
-```shell
-    systemd --version
-```
-
-In order to build the sink service, systemd headers are needed
-
-```shell
-    sudo apt install libsystemd-dev
-```
+The implementation uses system bus that has enforced security.
+In order to obtain a service name on system bus, the user launching the sink
+service must be previously declared to system.
+Provided file com.wirepas.sink.conf inside sink_service folder
+must be copied under /etc/dbus-1/system.d/ and edited with the user that will
+launch sink_service (and transport service).
 
-Python 3 and a recent pip version (>= 18.1)
+To change the default wirepas user, please edit the following lines
+from com.wirepas.sink.conf:
 
-```shell
-    sudo apt install libsystemd-dev python3 python3-dev python3-gi
-    wget https://bootstrap.pypa.io/get-pip.py \
-       && sudo python3 get-pip.py && rm get-pip.py \
-       && sudo pip3 install --upgrade pip
+```xml
+    <!-- Only wirepas user can own the service name -->
+    <policy user="wirepas">
 ```
 
-## Installation
+*It is recommended to restart your gateway once this file is copied.*
 
-### Sink service
+##### Option 1.1
```

Comment:
What do you think if we have a more explicit header? Such as

Option 1.1:  from source code

Reply:
Yes sure!